pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ParticipantUtil.java

```diff
@@ -110,4 +118,40 @@ public static ParticipantData findApplicantData(String ddpParticipantId, List<Pa
     public static boolean isLegacyAltPid(String participantId) {
         return !isGuid(participantId);
     }
+
+    public static String getParticipantGender(ElasticSearchParticipantDto participantByShortId, String realm, String ddpParticipantId) {
```

Comment:
On second thought, why not move this and dependent methods into ElasticSearchParticipantDto?  These methods are basically just checking internal state of that class.